repository/c.json

```diff
@@ -4400,6 +4400,17 @@
 				}
 			]
 		},
+		{
+	        "name": "CSS-Intellisense",
+	        "details": "https://github.com/id-sakurata/CSS-Intellisense",
+			"labels": ["css", "autocomplete", "intellisense"],
```

Comment:
```suggestion
		"labels": ["css", "autocomplete", "intellisense"],
```

Reply:
I made this plugin so that I can select class files according to what I need and don't have to have all class files in the autocomplete cache to reduce RAM usage. maybe this is useful for some people. In the settings I also include whether the user wants to activate the feature to search for classes automatically or not, there is also a quick action to add a folder to the cache by right clicking on the folder list sidebar. sorry my bad english, i can only use google translate.

Reply:
This is my first time doing a plugin publication request, so I don't know what to do.